lib/models/star-wars-movies.js

```diff
@@ -0,0 +1,44 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const schema = new Schema({
+  title: {
+    type: String,
+    required: true
+  },
+  episodeNumber: {
+    type: Number,
+    required: false,
+    min: 1,
+    max: 9
+  },
+  yearOfRelease: {
+    type: Number,
+    required: true
+  },
+  availableToWatch: [{
+    type: String,
+    enum: ['vhs', 'dvd', 'blueray', 'rent online', 'buy online'],
```

Comment:
I'm not going to take credit off since everything else looks really solid, but you should test the enum to make sure invald availableToWatch values are rejected